README.md

```diff
@@ -225,6 +225,7 @@ Nuxt.js is a framework for creating Universal Vue.js Applications.
 - [61 Financial News](https://news.61financial.com.au) - Australia financial news platform for Chinese readers, powered by Nuxt.js with best practices and well-tuned performance.
 - [OneConcern](https://oneconcern.com) ([source](https://github.com/acidjazz/oneconcern)) - Artifical Intelligence platform for Disasters.
 - [Pilot](https://pilot.com.au) - Online doctor consultation, Ecommerce Progressive Web Application made with Nuxt.js & Prisma.
+- [SUSHI](https://sushi.partners) - :sushi: Sutill Undergoing Severe Hassle with Irritating paperwork? = SUSHI, Document generator for establishing company in Japan.
```

Comment:
```suggestion
- [SUSHI](https://sushi.partners) - :sushi: Still Undergoing Severe Hassle with Irritating paperwork? = SUSHI, Document generator for establishing company in Japan.
```

Typo? :relaxed:

Reply:
@manniL Sorry..., Thank you for checking! I fix it.